### PR TITLE
chore: update nugetpackages and adapt change of Casbin.NET IAdapter interface 

### PR DESCRIPTION
**Pull request by ChangeTheCode**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@ChangeTheCode plz fix conflicts.

@Sagilio @xcaptain plz review.

File: EF-Adapter/CasbinDbAdapter.cs [hunk 1]
```diff
@@ -65,12 +82,26 @@ public void AddPolicy(string sec, string ptype, IList<string> rule)
 
         #region async operations
 
+        /// <inheritdoc />
+        public async Task AddPoliciesAsync(string sec, string ptype, IEnumerable<IList<string>> rules)
+        {
+            await Task.Run(() => AddPoliciesAsync(sec, ptype, rules));
```
Comment (xcaptain) [inline EF-Adapter/CasbinDbAdapter.cs]: Is it an infinite loop?

Comment (sagilio) [reply EF-Adapter/CasbinDbAdapter.cs]: @xcaptain Yes, the unit tests do not cover this API.

File: EF-Adapter/CasbinDbAdapter.cs [hunk 2]
```diff
@@ -20,17 +20,32 @@ public CasbinDbAdapter(CasbinDbContext context)
...
+        /// <inheritdoc />
         public void RemovePolicy(string sec, string ptype, IList<string> rule)
         {
             RemoveFilteredPolicy(sec, ptype, 0, rule.ToArray());
         }
 
+        /// <inheritdoc />
+        public void AddPolicies(string sec, string ptype, IEnumerable<IList<string>> rules)
+        {
+            Task.FromResult(AddPoliciesAsync(sec, ptype, rules));
```
Comment (sagilio) [inline EF-Adapter/CasbinDbAdapter.cs]: The difference between the sync and async method is to use SaveChanges or SaveChanegsAsync, we should not create unnecessary task instances.

**Review by sagilio — CHANGES_REQUESTED** [review]
There is no valid code in Batch APIs, just calling each other and no test coverage.

File: EF-Adapter/CasbinDbAdapter.cs (hunk 1 above)
Comment (ChangeTheCode) [reply EF-Adapter/CasbinDbAdapter.cs]: well discovered i will change that and create a test to increase the coverage

File: EF-Adapter/CasbinDbAdapter.cs [hunk 3]
```diff
@@ -36,12 +36,19 @@ public void RemovePolicy(string sec, string ptype, IList<string> rule)
         /// <inheritdoc />
         public void AddPolicies(string sec, string ptype, IEnumerable<IList<string>> rules)
         {
-            Task.FromResult(AddPoliciesAsync(sec, ptype, rules));
+            foreach (var rule in rules)
```
Comment (sagilio) [inline EF-Adapter/CasbinDbAdapter.cs:39]: Why not use `_context.CasbinRule.AddRange` like other batch APIs?

**Review by sagilio — APPROVED** [review]

**Comment by hsluoyz** [comment]
@ChangeTheCode the code quality is not satisfactory. Maybe you can just raise issues and let @sagilio check in the code.

**Pull request closed without merging**